hbase-oss/src/test/java/org/apache/hadoop/hbase/oss/sync/TestTreeLockManager.java

```diff
@@ -16,14 +16,14 @@
  * limitations under the License.
  */
 
-package org.apache.hadoop.hbase.oss;
+package org.apache.hadoop.hbase.oss.sync;
```

Comment:
Unrelated change, I can drop it if you like, but I found this test to be in the wrong package.

Reply:
Nice catch! I think it's fine to include it within this work.